### PR TITLE
Fixes #315 verify and cookies parameters in * On Session keywords

### LINKED ISSUE #315: GET On Session throws error "got multiple values for keyword argument 'verify'" when verify=${False} passed
Hi @lucagiove ,

The keyword "GET On Session" throws the following error if value for 'verify' is passed. Since I couldn't pass this optional parameter, I was not able to disable server's TLS certificate validation. 

TypeError: get() got multiple values for keyword argument 'verify'

`GET On Session   session        /auth   params=${get params}    headers=${headers}  allow_redirects=${false}   verify=${False}`

```
    def get_on_session(self, alias, url, params=None,
                       expected_status=None, msg=None, **kwargs):
```
| ``verify``  | Either a boolean, in which case it controls whether we verify the server's TLS certificate, or a string, in which case it must be a path to a CA bundle to use. Defaults to ``True``. |

Error Snip
----------
${resp} = RequestsLibrary . GET On Session session, /auth, params=${get params}, headers=${headers}, allow_redirects=${false}, verify=${False}
Documentation:	
Sends a GET request on a previously created HTTP Session.
Start / End / Elapsed:	20210202 16:51:59.030 / 20210202 16:51:59.032 / 00:00:00.002
16:51:59.030	TRACE	Arguments: [ 'session' | '/auth' | params={'client_id': 'test-app',
 'redirect_uri': 'http://127.0.0.1:5555/callback',
 'response_type': 'code',
 'scope': 'audience:server:client_id:test-app openid profile email offline_access',
 'state': 'test123'} | headers={'Host': 'xxxxx.local'} | allow_redirects=False | verify=False ]	
16:51:59.032	FAIL	TypeError: get() got multiple values for keyword argument 'verify'	
16:51:59.032	DEBUG	Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/RequestsLibrary/utils.py", line 138, in decorator
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/RequestsLibrary/RequestsOnSessionKeywords.py", line 49, in get_on_session
    params=params, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/RequestsLibrary/SessionKeywords.py", line 575, in _common_request
    **kwargs)	
16:51:59.024	TRACE	Arguments: [  ]

bash-4.2# pip list | grep req
requests                  2.25.1
requestsexceptions        1.4.0
robotframework-requests   0.8.0
bash-4.2#

Regards,
Sivaraman.R

### PR DESCRIPTION
**Pull request by lucagiove**

### CODE REVIEW

**Comment by lucagiove** [comment]
@sivaeric this should fix the Verify error

File: src/RequestsLibrary/SessionKeywords.py [hunk 1]
```diff
@@ -566,12 +565,15 @@ def _common_request(
         method_function = getattr(session, method)
         self._capture_output()
 
+        try:
+            kwargs['cookies']
+        except KeyError:
+            kwargs['cookies'] = self.cookies
+
```
Comment (NicoMincuzzi) [inline src/RequestsLibrary/SessionKeywords.py:572]: You can use `defaultdict` (https://docs.python.org/3/library/collections.html#collections.defaultdict) or `.get()` method with a default value than `try except`

**Review by NicoMincuzzi — COMMENTED** [review]
Improvement about dict

File: src/RequestsLibrary/SessionKeywords.py (hunk 1 above)
Comment (lucagiove) [reply src/RequestsLibrary/SessionKeywords.py:572]: Problem with .get is that I need to do this assignment only if cookies is not in the dictionary, and None might be a value... but while I'm writing a solution could be this

> # #
```suggestion
        self._capture_output()

        resp = method_function(
            self._get_url(session, uri),
            params=utils.utf8_urlencode(kwargs.pop('params', None)),
            timeout=self._get_timeout(kwargs.pop('timeout', None)),
            cookies=kwargs.pop('cookies', self.cookies),
            **kwargs)

        log.log_request(resp)
```

**Pull request merged**